Web/starter_project_barbies/utils/dateFormatter.ts

```diff
@@ -0,0 +1,4 @@
+export function formatDate(date : Date) {
```

Comment:
this is nice but it only formats in a specific format right ?
So if someone else were to use date formatting in a different format, they would use a different function

now the naming implies that this is the formatDate function and no other exists 
so either make it so that this does all kinds of formatting and you take in the format that people want to use, or make sure the function name implies what kind of format it does

Reply:
personally i would rather use prebuilt formatting libraries that take in the format (like moment js - nor sure if it is too big a library tho) instead of having my own util function